### ISSUE TITLE
How to normalize the data for 1D CFD ?

### ISSUE THREAD

**Issue by LouisSerrano** [pdebench/PDEBench#53]
Hi, as described in the paper, the range of values for the density and pressure is really different from the velocity. How would you recommend to normalize these channels ?

**Comment by LouisSerrano** [comment]
@mtakamoto-D Sorry for the tagging. I did not find explanations on how the 1D-CFD data can be pre-processed. What is the typical scale for the pressure and density across all the different 1D-CFD configurations ? How should use the data while making sure it makes sense both for the neural networks and for learning the dynamics ?

**Comment by mtakamoto-D** [comment]
Hi. In conclusion, we did not normalized CFD solution at all. 
This is because normalized solutions are not the solution of CFD anymore because of the non-linear nature of the equations. 
If you want to train your model in order to learn CFD equation rigorously (as neural operator), training data should be the raw solution (without normalization). If mathematical rigidity is not so important, any normalization might help DNN to learn some functional mapping corresponding with temporal update but the prediction may not learn CFD natures...

**Comment by LouisSerrano** [comment]
So you trained directly with an MSE loss ?

**Comment by kmario23** [comment]
Yes, as was noted on these lines:
[FNO loss - fno/train.py#L143](https://github.com/pdebench/PDEBench/blob/main/pdebench/models/fno/train.py#L143)
[UNet loss - unet/train.py#L152](https://github.com/pdebench/PDEBench/blob/main/pdebench/models/unet/train.py#L152)

Having said that, I can recommend you to try using a normalized L2 norm (aka. L2 Relative Error) as the loss, which is also very popular in the ML4Science community.

https://github.com/neuraloperator/neuraloperator/blob/main/neuralop/losses/data_losses.py#L130
